lib/plugin/instance.rb

```diff
@@ -588,11 +588,13 @@ def activate!
 
       # Automatically include all ES6 JS and hbs files
       root_path = "#{root_dir_name}/assets/javascripts"
+      DiscoursePluginRegistry.register_glob(root_path, 'js')
```

Comment:
Isn't this intentionally omitted? Should only be there with `transpile_js`, otherwise plugins that had extra js files in there will have them in the app.

Reply:
It should be `DiscoursePluginRegistry.register_glob(root_path, 'js') if transpile_js` I think.